packages/components/buttons/icon-button/src/icon-button.tsx

```diff
@@ -67,9 +92,9 @@ export type TIconButtonProps<
    */
   theme?: 'default' | 'primary' | 'info';
   /**
-   * Indicates the size of the icon.
+   * Indicates the size of the icon. Available sizes are '10', '20', '30', '40'.
    */
-  size?: 'small' | 'medium' | 'big';
+  size?: TLegacySizes | TSizes;
```

Comment:
Perhaps we can also use a warning within the updated components to let users know about the deprecated size values. 

[Here](https://github.com/commercetools/ui-kit/blob/main/packages/components/buttons/primary-button/src/primary-button.tsx#L101) is an example on how to use the `useWarning` hook

Reply:
I added it to all buttons. [IconButton](https://github.com/commercetools/ui-kit/pull/2830/files#diff-864898bbe411df91fd117fb7208de606c7d979c399fabb1cf5dbe90933fcff94R120), [PrimaryButton](https://github.com/commercetools/ui-kit/pull/2830/files#diff-3a56ecdd8c0ec240bd13af3c79568baf5b6c619a5b6e6443dacdae064f24a7a4R120), [SecondaryButton](https://github.com/commercetools/ui-kit/pull/2830/files#diff-c7dc6615dcde1e6e98272ffe5ee052955de20e77af64f177c5f0c78b9792acc0R153) and [SecondaryIconButton](https://github.com/commercetools/ui-kit/pull/2830/files#diff-ab4d692c622bc07c621c7eb40977e7f7ab012f23e7d0674d4944849b880f0319R93).